model/qti/templates/interactions/qti.portableCustomInteraction.tpl.php

```diff
@@ -1,46 +1,61 @@
 <?php
-/*
+
+/**
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
  * as published by the Free Software Foundation; under version 2
  * of the License (non-upgradable).
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- * 
- * Copyright (c) 2013 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
- *               
- * 
+ *
+ * Copyright (c) 2013-2023 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
  */
+
+declare(strict_types=1);
+
 ?>
+
 <customInteraction <?=get_data('attributes')?>>
-    <portableCustomInteraction customInteractionTypeIdentifier="<?=get_data('typeIdentifier')?>" hook="<?=get_data('entryPoint')?>" version="<?=get_data('version')?>" xmlns="http://www.imsglobal.org/xsd/portableCustomInteraction">
+    <portableCustomInteraction
+            xmlns="<?=get_data('xmlns')?>"
+            customInteractionTypeIdentifier="<?=get_data('typeIdentifier')?>"
+            hook="<?=get_data('entryPoint')?>"
+            version="<?=get_data('version')?>"
+    >
         <resources>
-            <?php if(count(get_data('libraries'))):?><libraries>
-                <?php foreach(get_data('libraries') as $lib):?>
-                <lib id="<?=$lib?>"/>
-                <?php endforeach;?>
-            </libraries><?php endif;?>
-            <?php if(count(get_data('stylesheets'))):?><stylesheets>
-                <?php foreach(get_data('stylesheets') as $stylesheet):?>
-                <link href="<?=$stylesheet?>" type="text/css" title="base"/>
-                <?php endforeach;?>
-            </stylesheets><?php endif;?>
-            <?php if(count(get_data('mediaFiles'))):?><mediaFiles>
-                <?php foreach(get_data('mediaFiles') as $file):?>
-                <file src="<?=$file?>"/>
-                <?php endforeach;?>
-            </mediaFiles><?php endif;?>
+            <?php if (count(get_data('libraries'))) :?>
```

Comment:
This refactoring may change line formatting in outcome xml making it less readable. 
If you changed it to avoid code format warnings in CI only, I would suggest adding `// phpcs:ignoreFile` comment